src/components/CreatePostContainer.js

```diff
@@ -0,0 +1,55 @@
+import React, { PureComponent } from 'react';
+import PostsForm from './PostsForm';
+import { connect } from 'react-redux';
+import { createPost } from '../actions/postActions';
+import PropTypes from 'prop-types';
+
+const mapDispatchToProps = dispatch => ({
+  onSubmit(title, body) {
+    dispatch(createPost(title, body));
+  }
+ 
+});
+
+
+class CreatePostContainer extends PureComponent{
+state={
+  titleText:'',
+  postText:'',
+}
+static propTypes = {
+  onSubmit: PropTypes.func.isRequired
+}
+
+
+handleSubmit = (e)=>{
+  e.preventDefault();
+  const { titleText, postText } = this.state;
+  this.props.onSubmit(titleText, postText);
+  this.setState({ 
+    titleText:'',
+    postText:''
+  });
+}
+
+
+
+handleTitleChange = ({ target })=>{
+  this.setState({ titleText:target.value });
+}
```

Comment:
you can reduce some code and create a app purpose handler here instead of having one for title and one for text